### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/deployment/cloud.md`

### PR DESCRIPTION
**Pull request by xzmeng**

### CODE REVIEW

File: docs/zh/docs/deployment/cloud.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# 在云上部署 FastAPI
+
+您几乎可以使用**任何云服务商**来部署 FastAPI 应用程序。
+
+在大多数情况下，主要的云服务商都有部署 FastAPI 的指南。
+
+## 云服务商 - 赞助商
+
+一些云服务商 ✨ [**sponsor FastAPI**](../help-fastapi.md#sponsor-the-author){.internal-link target=_blank} ✨，这确保了FastAPI 及其**生态系统** 持续健康地**发展**。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/cloud.md:9]:
```suggestion
一些云服务商 ✨ [**赞助 FastAPI**](../help-fastapi.md#sponsor-the-author){.internal-link target=_blank} ✨，这确保了FastAPI 及其**生态系统**持续健康地**发展**。
```

File: docs/zh/docs/deployment/cloud.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+这表明了他们对 FastAPI 及其**社区**（您）的真正承诺，因为他们不仅想为您提供**良好的服务**，而且还想确保您拥有一个**良好且健康的框架**、FastAPI。 🙇
```
Comment (socket-socket) [inline docs/zh/docs/deployment/cloud.md:11]:
```suggestion
这表明了他们对 FastAPI 及其**社区**（您）的真正承诺，因为他们不仅想为您提供**良好的服务**，而且还想确保您拥有一个**良好且健康的框架**，FastAPI。 🙇
```

**Review by socket-socket — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit cc4ec7bd1a12eba734a34cab50ab95fae9100b83 at: https://7f1d8f84.fastapitiangolo.pages.dev

File: docs/zh/docs/deployment/cloud.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+您几乎可以使用**任何云服务商**来部署 FastAPI 应用程序。
+
+在大多数情况下，主要的云服务商都有部署 FastAPI 的指南。
+
+## 云服务商 - 赞助商
+
+一些云服务商 ✨ [**赞助 FastAPI**](../help-fastapi.md#sponsor-the-author){.internal-link target=_blank} ✨，这确保了FastAPI 及其**生态系统**持续健康地**发展**。
+
+这表明了他们对 FastAPI 及其**社区**（您）的真正承诺，因为他们不仅想为您提供**良好的服务**，而且还想确保您拥有一个**良好且健康的框架**，FastAPI。 🙇
```
Comment (socket-socket) [inline docs/zh/docs/deployment/cloud.md:11]:
```suggestion
这表明了他们对 FastAPI 及其**社区**（您）的真正承诺，因为他们不仅想为您提供**良好的服务**，而且还想确保您拥有一个**良好且健康的框架**：FastAPI。 🙇
```

**Review by socket-socket — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 096ec916c2d414c287ba20fdd8acec205d4aed30 at: https://4ba51e6e.fastapitiangolo.pages.dev

**Review by shamosishen — APPROVED** [review]

**Review by Aruelius — APPROVED** [review]

**Comment by tiangolo** [comment]
Nice, thank you @xzmeng! 🍰 

And thanks for the reviews @shamosishen, @Aruelius, @socket-socket 🙇

**Pull request merged**